eval/eval.py

```diff
@@ -1914,9 +1963,7 @@ def token_results(
                                 wrong_span += 1
                                 errtypefreqs[xtype]["wrong_span"] += 1
                             # 3. Error correction
-                            # Get the 'corrected' attribute if available,
-                            # otherwise use ytok['suggest']
-                            ycorr = getattr(ytok, "corrected", ytok.suggest)
+                            ycorr = getattr(ytok, "suggest", ytok.suggest)
```

Comment:
The default here doesn't work - if `ytok` doesn't have the attribute `"suggest"`, it still tries to read `ytok.suggest`, which will fail.

Reply:
Fixed